README.md

```diff
@@ -0,0 +1,126 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> math_magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to: Make simple calculations and read a random math-related quote.
+
+## Built With
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Live Demo 
+
+Coming Soon
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
- [ ] Please add the following commands in this section :arrow_down:

```
git clone git@github.com:OmarMWarraich/math-magicians.git
cd math-magician
npm install
npm start
```
This is your readme file so please take it seriously :+1: